pennylane/estimator/ops/qubit/parametric_ops_single_qubit.py

```diff
@@ -49,8 +50,8 @@ def _rotation_resources(precision=1e-9):
         where each object represents a specific quantum gate and the number of times it appears
         in the decomposition.
     """
-    num_gates = round(1.149 * np.log2(1 / precision) + 9.2)
-    t = resource_rep(T)
+    num_gates = round(1.149 * math.log2(1 / precision) + 9.2)
```

Comment:
Why using math instead of numpy? Was it promoting it to being a numpy array instead a normal float? How was this surfaced by the adjoint and control changes?